### ISSUE TITLE
_fast_decode for multiple datashards

### ISSUE THREAD

**Issue by apeterswu** [tensorflow/tensor2tensor#502]
Hi, 

Is there any support to use _fast_decode for multiple datashards (gpu cards) if I want to sample data during training? I can only find slow_greedy_infer for multiple datashards currently, but it is too slow... Thanks.

Appreciate it very much if anyone can help.

**Comment by rsepassi** [comment]
I don't believe that's possible right now. We do have support in `t2t-decoder` for multiple shards (`--decode_shards` and `--worker_id`), but that's when you launch multiple instances of `t2t-decoder`.

**Comment by apeterswu** [comment]
Hope this feature can be added soon. Thanks.
